### ISSUE TITLE
Error when using mseloss

### ISSUE THREAD

**Issue by Yizhang-Li** [iancovert/shapley-regression#1]
I just transform True & False to 1&0 in census example. After training the model under lgb's rmse loss, I run the same code except that I replace the crossentropyloss with mseloss in the LossSHAP cooperative game. 

The error message is as follows.

![mse_error.png](https://i.loli.net/2021/04/05/WXmJLwCy5ejsIEl.png)

**Comment by iancovert** [comment]
Hi Yizhang, thanks for posting about this. There was a mistake in `mseloss` function, I incorrectly assumed that the predictions and labels would have shape `(batch, dim)`, but in this case it's just `(batch,)`. The error has been fixed [here](https://github.com/iancovert/shapley-regression/blob/master/shapreg/utils.py#L21).

Let me know if you come across any other issues!
